venom_core/api/routes/system_llm.py

```
@@ -2064,6 +2083,64 @@ async def get_llm_runtime_options():
+
+    Cel: 231C — separacja kolejki statusowej od inferencyjnej i obserwowalność
+    aktywnego runtime bez aktywnego sondowania nieaktywnych stosów.
+    """
+    runtime = get_active_llm_runtime()
+    orchestrator = system_deps.get_orchestrator()
+    queue_status: dict[str, Any] | None = None
+    queue_error: str | None = None
+    if orchestrator is not None:
+        try:
+            queue_status = orchestrator.get_queue_status()
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)